src/libraries/System.Net.Http/src/System/Net/Http/HttpClient.CreateDefaultHandler.tvOS.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Net.Http
+{
+    public partial class HttpClient
+    {
+        private const string HandlerSwitchName = "System.Net.Http.UseNativeHttpHandler";
+
+        private static HttpMessageHandler CreateDefaultHandler()
+        {
+            // Default is to use the Android native handler
+            if (!AppContext.TryGetSwitch(HandlerSwitchName, out bool isEnabled))
+            {
+                return new HttpClientHandler();
+            }
+
+            Type? runtimeOptions = Type.GetType("ObjCRuntime.RuntimeOptions, Xamarin.iOS");
```

Comment:
this should probably be `Xamarin.TVOS`:

```suggestion
            Type? runtimeOptions = Type.GetType("ObjCRuntime.RuntimeOptions, Xamarin.TVOS");
```